### ISSUE TITLE
Use SSH for clone-repos

### ISSUE THREAD

**Issue by amueller** [gitomator/gitomator#16]
I have an ssh key set up for github, and I also have a ``.gitomator`` file with an API key.
But gitomator-clone-repos uses https, which means I have to enter my password 100 times.
Is there a way to use ssh, preferably using the github ssh key I already installed?

**Comment by joeyfreund** [comment]
Sounds like #8 

At the moment, there is no way to select between `http` and `ssh`.
On my machine, I use SSH agent, to avoid entering my password every time.

If you already have your ssh key set up, you can run the command `ssh-add` (or, `ssh-add -K PATH-TO-PRIVATE-KEY`).
 > See [this page](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) for more details.

**Comment by amueller** [comment]
How do you use the ssh agent though? I have it set up and use ssh for all my github repos, but it's not used here.
It looks like gitomator calls ``git clone https://github.com/<org>/<repo>.git`` and that will always use https, right?

**Comment by joeyfreund** [comment]
Sorry, I might be confused, as it's been more than a year since I played around with the setup on y local machine.

Check out [this thread on StackOverflow](http://stackoverflow.com/questions/5343068/is-there-a-way-to-skip-password-typing-when-using-https-on-github), it explains how to avoid typing your credentials repeatedly when using HTTPS.

 > TL;DR `git config --global credential.helper cache` will remember your password (for 15 minutes by default), so you only have to type it once.

Please let me know if this workaround does the trick.

**Comment by amueller** [comment]
hm alright, but I'd prefer to just use ssh.

**Comment by joeyfreund** [comment]
@amueller , I just released an update that lets you specify the default protocol (`ssh` or `https`) in the `~/.gitomator` config file.

You can see the [example in the docs](https://gitomator.github.io/docs/quick-start/).

Can you please try to `gem install gitomator` and confirm that the issue is solved? Thanks
